src/exampleTypes/ResultTable/TableBody.js

```diff
@@ -4,6 +4,22 @@ import { observer } from 'mobx-react'
 import { getRecord, getResults } from '../../utils/schema'
 import HighlightedColumn from './HighlightedColumn'
 
+let blankText = _.flow(
+  _.toString,
+  _.map(x => (x === ' ' ? x : '\u2588')),
+  _.join('')
+)
+
+let blankResult = (display, data, record) => {
+  let formatted = display(data, record)
+  if (typeof formatted === 'object') {
+    if (_.isArray(_.get('props.children', formatted))) return null
+    else return display(blankText(data), record)
```

Comment:
why do we care what the display was before?

Reply:
Sometimes `display` returns React children, so in this case, I run `display` again passing it `blankText(data)`. If `display` returns text I just make it blank. I am going to add comments to make it notable